src/main/java/com/kodilla/ecommercee/UserController.java

```diff
@@ -10,26 +9,20 @@
 public class UserController {
     private long key;
 
-    private UserMapper userMapper;
-
-    @Autowired
-    public UserController(UserMapper userMapper) {
-        this.userMapper = userMapper;
-    }
-
     @PostMapping("createUser")
-    public User createUser(@RequestBody UserDto userDto) {
-        return userMapper.mapToUser(userDto);
+    public UserDto createUser(@RequestBody UserDto userDto) {
+        return userDto;
     }
 
-    @GetMapping("banUser")
-    public void banUser(Long userId) {
-
+    @PutMapping("banUser")
+    public UserDto banUser(@RequestBody UserDto userDto) {
+        return new UserDto(userDto.getId(), userDto.getUsername(), "-1", userDto.getUserKey());
     }
 
     @GetMapping("generateKey")
-    public Long generateKey() {
-        return key = new Random().nextLong();
+    public String generateKey(@RequestParam long userId) {
+        key = new Random().nextLong();
+        return String.valueOf(key + userId);
```

Comment:
jak napiszesz
return key + userId; 
to chyba też zadziała? 
Jaką mamy wartość dodaną z użycia String.valueOf ?

Reply:
Poprawiłem, czy aktualna wersja jest okej? Mój kolejny commit również nazywa się podejście nr 3 bo zapomniałem zmienić message do commita.